js/views/callinfoview.js

```diff
@@ -66,13 +66,33 @@
 		'		{{/if}}' +
 		'	</div>' +
 		'{{/if}}' +
-		'{{#if showShareLink}}' +
-		'	<div class="share-link-options">' +
-		'		<div class="clipboard-button"><span class="button icon-clippy"></span></div>' +
-		'	</div>' +
-		'{{/if}}' +
+		'<div class="talk-settings-button">' +
+		'		<span class="button icon-settings"></span>' +
+		'</div>' +
+		'</div>' +
+		'<div class="settings-menu hidden">' +
```

Comment:
I'm not sure this should be in the callinfo-section, as it is not specific to the conversation, but in general?

Maybe it's time for us to start using the settings feature in the left sidebar. However I'm not sure we should go through the trouble, when we can do this easily after switching to Vue.JS ( #1347 )

Reply:
I think it is specific to the conversation just like the 'join call' button, the settings are used directly when joining the call and affect how the call appears and sounds.